WhaleSpotting/react-app/src/styles/Navbar.scss

```diff
@@ -95,6 +96,7 @@
     @media(max-width: $mobileScreenSize) {
 
         justify-content: left;
+        padding-left: 5px;
```

Comment:
Was this purposeful?

Reply:
We think we need it. This allows space from the left edge of the screen.

With padding-left:
![image](https://user-images.githubusercontent.com/67764574/128194992-0a4496a9-be8d-4e89-9130-1aa89414d086.png)

Without padding-left:
![image](https://user-images.githubusercontent.com/67764574/128195075-e7274d46-1663-424b-9df6-2b8922bd5017.png)

Reply:
as per your comment, remove padding here, added margin on the current page. Done.